README.md

```diff
@@ -593,6 +593,74 @@ flat_map                 13.64 MB - 1.67x memory usage
 **All measurements for memory usage were the same**
 ```
 
+#### Generating a random number with `Enum.rand/1` and `:random.unform/1` [code](code/general/enum_random_vs_rand_uniform.exs)
```

Comment:
What about other ways of generating a random number, like just `random:uniform()` without the limit? If we're looking at just a random number generator, there are a lot of ways to go about that that don't specify the range the number must be in. For example, there's also Erlang's `rand` module, which uses a different algorithm than `random` but isn't truly random enough for stuff like cryptography, but might be random enough for most other uses.

Reply:
What I needed was to generate a random number among a range. I should've explicitly explained that in the description.

What do you want to do? I can expand this, change the description to try explain that it is generating a number between a range/limit, or just close the PR if you think it is an example that it is too broad. No hard feelings :)